solidity/contracts/AugurConstantProductMarketRouter.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.29;
+
+import { IERC1155 } from "./IERC1155.sol";
+import { IERC20 } from "./IERC20.sol";
+import { IShareToken } from "./IShareToken.sol";
+import { IMarket } from "./IMarket.sol";
+import { IAugur } from "./IAugur.sol";
+import { IAugurConstantProduct } from "./IAugurConstantProduct.sol";
+import { Constants } from "./Constants.sol";
+
+contract AugurConstantProductRouter {
+
+	IShareToken shareToken = IShareToken(Constants.SHARE_TOKEN);
+	IERC20 public dai = IERC20(Constants.DAI_ADDRESS);
+	IAugur public constant augur = IAugur(Constants.AUGUR_ADDRESS);
+	uint256 private constant numTicks = 1000;
```

Comment:
is this always 1000 for yes no markets?

Reply:
Yea! I had forgotten we just hardcoded it in Augur but apparently thats the case